README.md

```diff
@@ -12,28 +12,61 @@ Enables AWS Config and adds managed config rules with good defaults.
 
 * approved-amis-by-tag: Checks whether running instances are using specified AMIs.
 
+### Backup Plan
+
+* backup-plan-min-frequency-and-min-retention-check: Checks if a backup plan has a backup rule that satisfies the required frequency and retention period. *No* *GOVcloud*
+
 ### CloudTrail
 
 * cloudtrail-enabled: Ensure CloudTrail is enabled.
 * cloud-trail-encryption-enabled: Ensure CloudTrail is configured to use server side encryption (SSE) with AWS KMS or CMK encryption.
 * cloud-trail-log-file-validation-enabled: Checks whether AWS CloudTrail creates a signed digest file with logs.
 * multi-region-cloud-trail-enabled: Ensure that there is at least one multi-region AWS CloudTrail enabled.
+* cloud-trail-cloud-watch-logs-enabled: Checks whether AWS CloudTrail trails are configured to send logs to Amazon CloudWatch logs.
 
 ### CloudWatch Logs
 
 * cloudwatch-log-group-encryption: Ensure that CloudWatch Logs are encrypted.
+* cw_loggroup_retention_period_check: Checks whether Amazon CloudWatch LogGroup retention period is set to specific number of days.
```

Comment:
Above you spell out `cloudwatch` but here you have `cw`. Some consistency in convention would be preferred.

Reply:
The first part of the description is the actual rule name in AWS Dev guide. I wouldn't want to change that for easy lookup in AWS. Admittedly, in this example, it seems the rule `cloudwatch-log-group-encryption` may actually have been renamed to `cloudwatch-log-group-encrypted` 

Also my mistaken use of underscores on this one.